db/seeds.rb

```diff
@@ -5,3 +5,52 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# users
+
+admin = [true, true, true, false, false, false, false, false, false, false, false, false, false, false, false]
+first_name = ["Peter", "Veronika", "Thomas", "Karl", "Morgan", "Tim", "Marita", "Bernhard", "Bianca", "Sia", "Claudio", "Laura", "Marie", "Luis", "Alex", "Manfred", "Alissa", "Ken", "Sebastian", "Zita"]
+last_name = ["Hill", "Landberg", "Hintermeyer", "Leoni", "Richardson", "Lagerfeld", "Arana", "Apmann", "Arco", "Sadi", "Tiedemann", "Wittelberg", "Neisemeier", "Iarossi", "Jenkara", "Scherhausen", "Linkhammer", "Ginser", "Patoli", "Dahme"]
+additional_infos = ["Hey there - your cardiologist is here.", "I am new to the team - whoop, whoop.", "Busy", "📈🩺👨🏼‍⚕️", "Saving some babies! #neonatal", "I want icecream 🍦",  "👩🏼‍⚕️", "In the gym", "Do not disturb - not taking any offers🤪", "Nihil since causa", "29.05.2022", "💙", "Passionate delivery nurse #lovemyjob", "All these christmas lights 🎶🎵", "tired😴", "Hey there!", "Pregnant - so excited🤰🏼", "Caught Covid 😥", "Looking forward to my vacantions." "I got a new pair of shoes😎"]
+experience_level = ["experienced", "experienced", "experienced", "beginner", "beginner", "experienced", "beginner", "experienced", "beginner", "experienced", "beginner", "experienced", "beginner", "experienced", "beginner", "beginner", "experienced", "beginner", "experienced", "experienced"]
+# functional_role
+names1 = ["nurse manager", "nurse manager", "nurse manager", "radiology nurse", "labor & delivery nurse", "emergency-room nurse", "intensive-care-unit nurse", "radiology nurse", "labor & delivery nurse", "emergency-room nurse", "intensive-care-unit nurse", "radiology nurse", "labor & delivery nurse", "emergency-room nurse", "intensive-care-unit nurse", "radiology nurse", "labor & delivery nurse", "emergency-room nurse", "intensive-care-unit nurse", "radiology nurse"]
+# preferred_slots
+weekday = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+# slots
+names2 = ["early", "late", "night", "early", "late", "night", "early", "late", "night", "early", "late", "night", "early", "late", "night", "early", "late", "night", "early", "late"]
+# date
+dates = ["Dec 6, 21", "Dec 7, 21", "Dec 8, 21", "Dec 9, 21", "Dec 10, 21", "Dec 11, 21", "Dec 12, 21", "Dec 13, 21", "Dec 14, 21", "Dec 15, 21", "Dec 16, 21", "Dec 17, 21", "Dec 18, 21", "Dec 19, 21", "Dec 20, 21", "Dec 21, 21", "Dec 22, 21", "Dec 23, 21", "Dec 24, 21", "Dec 25, 21"]
+# user_shifts
+status = ["open", "assigned", "open", "assigned", "open", "assigned", "open", "assigned", "open", "open", "assigned", "open", "assigned", "open", "assigned", "open", "assigned", "open"]
```

Comment:
don't we also have unassigned shifts?
Or is that why there are only 18 items?

Reply:
changed it to 20 - we have only open and assigned; happy to discuss in the team again.